### ISSUE TITLE
If `gptscript credentials` is executed even before any script execution is done on fresh installs , it fails with error - gptscript-credential-osxkeychain: no such file or directory, out

### ISSUE THREAD

**Issue by sangee2004** [gptscript-ai/gptscript#595]
gptscript version - v0.8.5+293f6f8a 

**Step to reproduce the problem:**

1. Install gptscript (Before installing gptscript , make sure there no `chache` (~/Library/Caches/)  and `config` (~/Library/Application Support) directory for gptscript application) 
2. Try to list credentials

This errors out:
```
 gptscript credentials                   
2024/07/01 11:12:08 failed to list credentials: error listing credentials - err: fork/exec /Users/sangeethahariharan/Library/Caches/gptscript/repos/gptscript-credential-helpers/bin/gptscript-credential-osxkeychain: no such file or directory, out: ``
```

**Note:**
gptscript script execution succeeds as expected in this case. Once script execution succeeds, we are able to list credentials as expected.

**Comment by g-linville** [comment]
@sangee2004 this should be fixed in 0.9.2

**Comment by sangee2004** [comment]
Tested with `0.9.2` . This issue is not seen anymore.
Credential helper gets built in this case.

```
gptscript credentials
14:21:51 Cloning https://github.com/gptscript-ai/gptscript-credential-helpers.git
14:21:54 Fetching 980c5f8eef5027f25e44919687f5764b46905714 at https://github.com/gptscript-ai/gptscript-credential-helpers.git
14:21:54 Checking out 980c5f8eef5027f25e44919687f5764b46905714 to /Users/sangeethahariharan/Library/Caches/gptscript/repos/gptscript-credential-helpers/repo/980c5f8eef5027f25e44919687f5764b46905714
14:21:54 Downloading Go 1.22.1
14:21:59 Building credential helper osxkeychain
```
